cmd/scollector/collectors/ifstat_linux.go

```diff
@@ -106,3 +112,162 @@ func c_ifstat_linux() (opentsdb.MultiDataPoint, error) {
 	})
 	return md, err
 }
+
+const (
+	linuxNetBondSlaveIsUpDesc = "The status of the bonded or teamed interface."
+	linuxNetBondSlaveCount    = "The number of slaves on the bonded or teamed interface."
+)
+
+func c_if_bond_linux() (opentsdb.MultiDataPoint, error) {
+	var md opentsdb.MultiDataPoint
+	const bondingPath = "/proc/net/bonding"
+	bondDevices, _ := ioutil.ReadDir(bondingPath)
+	for _, fi := range bondDevices {
+		var iface string
+		var slaveCount int
+		if err := readLine(bondingPath+"/"+fi.Name(), func(s string) error {
```

Comment:
`filepath.Join`. If you ever type a `/` as a path seperator you should feel bad.